src/domains/citizen-auth-app/04_fims.tf

```diff
@@ -52,8 +52,8 @@ locals {
       MONGODB_URL                   = data.azurerm_key_vault_secret.mongodb_connection_string_fims.value
       AUTHENTICATION_COOKIE_KEY     = "X-IO-FIMS-Token"
       GRANT_TTL_IN_SECONDS          = "86400"
-      ISSUER                        = "http://localhost:3001"     #TBD with domain value
-      COOKIES_KEY                   = "just-for-testing-purposes" #TBD with vault value
+      ISSUER                        = "https://io-p-citizen-auth-weu-prod01-app-fims.azurewebsites.net"
+      COOKIES_KEY                   = "whatasecret" #TBD with vault value
```

Comment:
We are changing this value that is always a test value. Can we set this from this PR inside a secret?.

Reply:
not now. We need some more tests before.